modules/thermal_hydraulics/test/tests/junctions/wcnsfv/two_components_same_vars.i

```diff
@@ -0,0 +1,140 @@
+# Operating conditions
```

Comment:
Is there no test for this input file?

Reply:
lemme check

Reply:
deleting it

Reply:
looking at a griffin issue then I ll reupload